src/scripts/shop/ShopHandler.ts

```diff
@@ -61,6 +66,16 @@ class ShopHandler {
         input.val(newVal > 1 ? newVal : 1).change();
     }
 
+    public static ownKeyItem(name: string): boolean {
+
+        console.log(name);
+        if(name =='Dungeon_ticket' && player.hasKeyItem('Dungeon ticket')){
```

Comment:
Probably better to just check player.hasKeyItem and then replace the underscore with a space

Reply:
isha said no issue with a hard coded fix for just dungeon ticket. if we change it to be dynamic - it will return false for non key item items. so I opted to first check and verify then item in question is dungeon_ticket and if so - also check if they own it. Not a permanent deal - but needed to get progress since i currently do not know a way to check if an item is a key item or not.